src/main/java/Bye.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Ends the ChatBot.
+ */
+public class Bye extends Command{
```

Comment:
Should there be a whitespace before the braces? Similar instances in other classes.